core/container/src/main/java/org/wildfly/swarm/internal/ArtifactManager.java

```diff
@@ -131,12 +134,13 @@ private File findFile(String gav) throws IOException, ModuleLoadException {
         }
 
         if (parts.length == 4) {
-            packaging = parts[2];
+            // the type "test-jar" is packaged as a .jar file more info: https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html
+            packaging = TEST_JAR.equals(parts[2]) ? JAR : TEST_JAR;
```

Comment:
Should the test be changed to:
`packaging = TEST_JAR.equals(parts[2]) ? JAR : parts[2]`

At present it looks like any packaging other than `test-jar` will result in the packaging being set to `test-jar`, which doesn't seem right

Reply:
Thanks @kenfinnigan  , sorry , I thought that but I wrote something else. 

One thing that I forgot to mention, the unit tests of this class are all commented, is there something wrong wit them?